### ISSUE TITLE
[flagd-web-provider] ERROR: Cannot find module './schema_pb.js' or its corresponding type declarations.

### ISSUE THREAD

**Issue by doocaat** [open-feature/js-sdk-contrib#688]
Hello everyone! I have a problem with typescript type.
I try user npm library @openfeature/flagd-web-provider in angular application but when I am importing 
`import {FlagdWebProvider} from '@openfeature/flagd-web-provider';`

I have this error:

```
Error: node_modules/@openfeature/flagd-web-provider/src/proto/ts/schema/v1/schema_connect.d.ts:1:315 
error TS2307: Cannot find module './schema_pb.js' or its corresponding type declarations.

import { EventStreamRequest, EventStreamResponse, ResolveAllRequest, ResolveAllResponse, ResolveBooleanRequest, ResolveBooleanResponse, ResolveFloatRequest, ResolveFloatResponse, ResolveIntRequest, ResolveIntResponse, ResolveObjectRequest, ResolveObjectResponse, ResolveStringRequest, ResolveStringResponse } from "./schema_pb.js";
```

**Comment by toddbaert** [comment]
Hey @doocaat . I'm looking into this. I can't fully reproduce it, but I think I have some ideas about what's happening.

As a temporary workaround, I'm 99% sure if you add [skipLibCheck](https://www.typescriptlang.org/tsconfig#skipLibCheck) to your tsConfig and bypass this with relatively low risk. That's not really a proper fix though, so I'll keep investigating.

**Comment by toddbaert** [comment]
Hey @doocaat ... I've released [0.4.1](https://github.com/open-feature/js-sdk-contrib/releases/tag/flagd-web-provider-v0.4.1) which I think should fix this. Please confirm.
